src/Process/FeatureProcessing.js

```diff
@@ -104,6 +104,17 @@ export default {
             // if request return empty json, WFSProvider.getFeatures return undefined
             result = result[0];
             if (result) {
+                // special case for FileSource, as it is not tiled and we need
+                // to attach it to the correct node
+                if (layer.source && layer.source.isFileSource) {
+                    for (const extentSrc of extentsSource) {
```

Comment:
could we use the global extent of `parsedData`?

Reply:
It is the case below ? I don't understand sorry

Reply:
Factorized